web_src/js/components/RepoActionView.vue

```diff
@@ -663,13 +665,23 @@ export function ansiLogToHTML(line) {
 
 .action-view-right {
   flex: 1;
-  color: var(--color-secondary-dark-3);
+  color: var(--color-console-fg-secondary);
   max-height: 100%;
   width: 70%;
   display: flex;
   flex-direction: column;
 }
 
+.action-view-right button {
+  background: transparent;
+  border: none;
+  color: var(--color-console-fg-secondary);
+}
+
+.action-view-right button:hover {
+  color: var(--color-console-fg);
+}
```

Comment:
I think these styles should be provided by #24989 , instead of duplicating these code again.

Reply:
Yes, I would probably use one style from that PR but I don't want this to depend on that PR now, so let's merge this and do style corrections in the other PR.

Reply:
`button-ghost` may fit here, thought I do kind of prefer it without that background highlight. I will in any case require some color overrides because this is the one exception in the UI where light theme renders dark.

Reply:
Please provide a complete framework level support, instead of "fine tuning" everything ad-hoc, or keeping telling developers to do this or do that.

Reply:
Not everything can be solved on "framework" level because special situations like this black box require special solutions. That said, let me try using a button-ghost, it will need color overrides of course because of this "non-standard" box.

Reply:
It doesn't look good to me, while I won't block.

Reply:
Well if your persist, I can shoehorn the current `button-ghost` into this.

Reply:
Done in https://github.com/go-gitea/gitea/pull/24876/commits/d184fe2a856fed4e7c92940da618a6d9e9bc77f9.

<img width="248" alt="Screenshot 2023-05-30 at 21 32 46" src="https://github.com/go-gitea/gitea/assets/115237/967814c0-7a26-41d8-88c7-58914d2e6a5b">

Reply:
I do not think `button-ghost` is right here because of the "hover background" problem. So I didn't mean anything must be changed here.

I meant "it doesn't look good to me, while the UI doesn't really bother me, I only worry about how could the future developers maintain these style code", and I didn't mean I have objection, so no block from my side.

ps: I believe a framework should provide intuitive supports to make developers could write code smoothly, and reduce unclear choices or ad-hoc patching.

Reply:
Happy with it now.

Yes, a framework can help in a lot of cases, but our color system currently only works when the background is light on light theme and dark on dark theme, which is the case here, necessiting these overrides.

I do have a few ideas to improve this and will do so in the button PR, but this console box will have to keep the exceptions. I can not magically read out the background color a button is rendered on.

Reply:
Thank you. Hope my comments don't cause misunderstanding. Sometimes I just express my personal opinion and maybe I am not able to find some suitable words. "no block" means it doesn't really bother me.

Reply:
I guess at some point we can rip out the fomantic button module after which we can design a proper button system. Will be better than layering all these hacks on top of fomantic.